aws/ec2_test.go

```diff
@@ -93,6 +93,16 @@ func createTestEC2Instance(t *testing.T, session *session.Session, name string,
 	return *runResult.Instances[0]
 }
 
+func removeEC2InstanceProtection(svc *ec2.EC2, instance *ec2.Instance) {
+	// make instance unprotected so it can be cleaned up
+	svc.ModifyInstanceAttribute(&ec2.ModifyInstanceAttributeInput{
```

Comment:
I'm guessing this method can return an error. You should check for it and fail the test if it does.